src/helpers/uriHandlerController.ts

```diff
@@ -2,34 +2,75 @@ import {startDebugging} from '@/commands';
 import {Constants} from '@/Constants';
 import {Uri, UriHandler, window} from 'vscode';
 
+/**
+ * This type of URI handler is used to handle the `truffle-vscode` URI scheme.
+ */
 type TDebugInformation = {
+  /**
+   * The transaction hash to debug.
+   */
   txHash: string;
+
+  /**
+   * The working directory of the project.
+   */
   workingDirectory: string;
+
+  /**
+   * The network provider url.
+   */
   providerUrl: string;
+
+  /**
+   * The fetch external contracts flag.
+   */
+  fetchExternal: boolean;
```

Comment:
I see there is some overlap between this type and `debuggerTypes.ILaunchRequestArguments`. Should we find a way to merge/reuse these definitions?

Reply:
let me check this one

Reply:
I've merged them, thanks! :)